packages/server/src/fhir/operations/export.ts

```diff
@@ -47,27 +47,33 @@ export async function bulkExportHandler(req: Request, res: Response): Promise<vo
 async function exportResourceType(exporter: BulkExporter, project: Project, resourceType: ResourceType): Promise<void> {
   const repo = exporter.repo;
   const hasMore = true;
+  let offset = 0;
   while (hasMore) {
     const bundle = await repo.search({
       resourceType,
       count: 1000,
+      offset,
```

Comment:
addresses infinite loop when there are more than 1000 resources for a resource type